### ISSUE TITLE
The method XXX couldn't be found in the compiled CloudFormation template

### ISSUE THREAD

**Issue by GerardPaligot** [DianaIonita/serverless-api-gateway-caching#24]
When I create a lambda with an underscore in its path:

```
get-linked-products-by-model-id:
  handler: models/handler.getLinkedProductsByModelId
  events:
    - http:
        path: /v1/models/{model_id}/linked
        method: get
        private: true
```

The generated CloudFormation method is `ApiGatewayMethodV1ModelsModelidVarLinkedGet` but this plugin try to find `ApiGatewayMethodV1ModelsModel_idVarLinkedGet`.

**Comment by DianaIonita** [comment]
Hi @GerardPaligot,

Thanks for spotting this. I've released [v1.2.2](https://github.com/DianaIonita/serverless-api-gateway-caching/releases/tag/1.2.2) which should fix this issue.
Let me know if you still have problems.

**Comment by GerardPaligot** [comment]
This bug is well fixed by the latest release. Thanks.
